source/docs/articles/local/drupal-drush-command-line-utility.md

```diff
@@ -8,39 +8,20 @@ keywords: Drupal drush, command line, drupal, terminus drush, cli
 ---
 [Drush](http://drush.org) is a command-line interface for Drupal that provides a wide set of utilities for administering and maintaining your site.
 
-Pantheon does not need the settings.php for your site to work, and your Drupal sites do not contain one out of the box. Drush commands require a settings.php file and it's considered a best practice to have one. Simply duplicate the `sites/default/default.settings.php` to `sites/default/settings.php` for Drush to work on a new site.
+Pantheon does not need the settings.php for your site to work, and your Drupal sites do not contain one out of the box. Drush commands require a settings.php file and it's considered a best practice to have one. Simply copy the `sites/default/default.settings.php` to `sites/default/settings.php` for Drush to work on a new site.

```

Comment:
Do we want to say anything about the fact that Drupal 8 sites **do** come with a bundled settings.php? Also, the bundled settings.php and default.settings.php both include the necessary settings.pantheon.inc. If someone used a stock default.settings.php, their Drupal 8 site would stop working. Maybe we should have another document that describes settings.php for Drupal 8 (did not check to see if that already exists), and just cross-link to it here?